tox_encryptsave/Cargo.toml

```diff
@@ -21,4 +21,8 @@ edition = "2018"
 [dependencies]
 tox_crypto = { version = "0.1.0", path = "../tox_crypto" }
```

Comment:
no need to keep tox_crypto as dep anymore?

Reply:
Yes, fixed.